crates/pathfinder/src/storage/schema/revision_0001.rs

```diff
@@ -0,0 +1,64 @@
+use rusqlite::Transaction;
+
+pub(crate) fn migrate_to_1(transaction: &Transaction) -> anyhow::Result<()> {
+    transaction.execute(
+        r"CREATE TABLE contract_code (
+            hash       BLOB PRIMARY KEY,
+            bytecode   BLOB,
+            abi        BLOB,
+            definition BLOB
+        )",
+        [],
+    )?;
+    transaction.execute(
+        r"CREATE TABLE contracts (
+            address    BLOB PRIMARY KEY,
+            hash       BLOB NOT NULL,
+
+            FOREIGN KEY(hash) REFERENCES contract_code(hash)
+        )",
+        [],
+    )?;
+    transaction.execute(
+        r"CREATE TABLE ethereum_blocks (
+            hash   BLOB PRIMARY KEY,
+            number INTEGER NOT NULL
+        )",
+        [],
+    )?;
+    // TODO: consider ON DELETE CASCADE when we start cleaning up. Don't forget to document if we use it.
```

Comment:
Since we are now doing the migrations, we should probably delete these comments to not cause confusion.
```suggestion
    // TODO: consider ON DELETE CASCADE when we start cleaning up. Don't forget to document if we use it.
```